challenges/challenge-weather-app/assets/script.js

```diff
@@ -0,0 +1,139 @@
+// API URLs and Keys
+const API_WEATHER = "https://api.openweathermap.org/data/2.5/weather";
+const API_UNSPLASH = "https://api.unsplash.com/search/photos";
+const WEATHER_KEY = "2af3ef06b75eef48a72ed617a00dad79"; // Replace with your OpenWeather API Key
+const UNSPLASH_KEY = "LxKeC1occOPs3F2utkGVSjjUSb0Irq6mzvzWuG7vNS0"; // Replace with your Unsplash Access Key
+
+// DOM Elements
+const photoEl = document.getElementById("photo");
+const thumbsEl = document.getElementById("thumbs");
+const conditionsEl = document.getElementById("conditions");
+const creditUserEl = document.getElementById("credit-user");
+const creditPlatformEl = document.getElementById("credit-platform");
+const searchForm = document.getElementById("search");
+const searchInput = document.getElementById("search-tf");
+
+// Default city
+let currentCity = "London";
+
+// Fetch weather data
+async function fetchWeather(city) {
+  const url = `${API_WEATHER}?q=${city}&appid=${WEATHER_KEY}&units=metric`;
+  try {
+    console.log(`Fetching weather data for: ${city}`);
+    const response = await fetch(url);
+    if (!response.ok) throw new Error(`City "${city}" not found`);
+    const data = await response.json();
+    console.log("Weather Data:", data);
+    return data;
+  } catch (error) {
+    console.error("Error fetching weather data:", error);
```

Comment:
Real users don't ever look in the console, so they'll never see this log line. What will they see? What could it be useful to do instead?

(This same question applies to other places you do this, too)